java/src/jmri/jmrit/logixng/actions/ActionBundle.properties

```diff
@@ -365,6 +365,9 @@ ShowDialog_Short                            = Show dialog
 ShowDialog_Long_TextOnly                    = Show dialog: Only text: {0}
 ShowDialog_Long_CommaSeparatedList          = Show dialog: Comma separated list
 ShowDialog_Long_StringFormat                = Show dialog: String.format: {0}
+# Important! These strings MUST be valid female socket names!!
```

Comment:
Should this mention these should not be L8N'd? Or say they should be, if that's the case?

Reply:
Thanks for asking.

They can be translated, but the name must follow the naming rules for an identifier. The name must start with a letter and it can contain letters, digits and underscore. PR #10484 adds validation for all locales to ensure the translated names are valid. The PR does that by creating an instance of every LogixNG action and expression, which in turn creates all the female sockets with the translated names. If the translated name is not a valid identifier, this test will fail.

The reason to why the name must be a valid identifier is that socket names can be used in formulas. And therefore the names must be parsable as identifiers.

```
       7:                      Digital Formula: result = "Train number is " + Tåg
       8:                          ?* Tåg
       9:                              Get memory IMTåg as string value
```

This formula uses the value from the child, using the socket name `Tåg` as an identifier in the formula.

`Tåg` is the Swedish word for `Train`. `Tåg` is a valid identifier, but `the train` is not a valid identifier since it contains a space.